src/components/templates/ContentTemplate.tsx

```diff
@@ -22,6 +22,8 @@ import QuizAnswerCard from '../organisms/QuizAnswerCard'
 import { SpacingText } from '../../utils/SpacingText'
 import Link from 'next/link'
 import { SVGS } from '../../constants/svgs'
+import BookMark from '../atoms/BookMark'
+import BookMarkListItem from '../molecules/BookMarkListItem'
 
 /**
  * exist : 질문 한개라도 이미 답변 한 상태
```

Comment:
How about making the type?

`export type qnaStatus = 'exist' | 'request'`

Reply:
great opinion